tools/filters/gff/gff_filter_by_feature_count.py

```diff
@@ -155,7 +155,7 @@ def __main__():
                 count += 1
         eval_text = '%s %s' % ( count, condition )
         if not check_expression(eval_text):
-            print("Invalid condition: %s, cannot filter." % condition, file=sys.stderr)
+            print "Invalid condition: %s, cannot filter." % condition, file=sys.stderr
```

Comment:
Yup - your right as always - I've taken two really poor swings at this. I'll try another.